src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -315,6 +322,12 @@ public async Task<object> GetDescriptionAsync(IAsyncCompletionSession session, V
                 attributeIcons: attributeImages);
 
             item.Properties.AddProperty(RoslynItem, roslynItem);
+
+            if (roslynItem.UseEditorCompletionItemCache)
```

Comment:
the bool seems unnecessary.  just always store this.  in the non-type-import case, the CompletionItems (and VSCompletoinItems) will just get GC'ed.  in the type-import case, it will hold the item around.  Basically, this means that anyone who wants to cache CompletionItems gets this benefit.

Reply:
fixed